### PR TITLE
local task qc

### PR DESCRIPTION
**Pull request by oliche**

### CODE REVIEW

File: task_qc_viewer/README.md [hunk 1]
```diff
@@ -25,12 +22,20 @@ Launch the Viewer by typing `ipython task_qc.py session_UUID` , example:
 ```
 ipython task_qc.py c9fec76e-7a20-4da4-93ad-04510a89473b
```
Comment (GaelleChapuis) [inline task_qc_viewer/README.md:23]: This example does not work

Comment (GaelleChapuis) [reply task_qc_viewer/README.md:23]: This one works: task_qc_viewer gaelle$ ipython task_qc.py fece187f-b47f-4870-a1d6-619afe942a7d

Comment (GaelleChapuis) [reply task_qc_viewer/README.md:23]: 2020-10-09 15:49:58.580 INFO     [task_metrics.py:72] Session /Users/gaelle/Downloads/FlatIron/zadorlab/Subjects/CSH_ZAD_029/2020-09-17/001: **Running QC on behavior data...**
This step takes ages. (minutes)

File: task_qc_viewer/task_qc.py [hunk 2]
```diff
@@ -11,24 +11,53 @@
...
         super().__init__(session_path, one=one, log=_logger)
-        self.load_data(bpod_only=bpod_only)
-        self.compute()
+
+        if local:
+            dsets, out_files = ephys_fpga.extract_all(session_path, save=True)
+            self.extractor = TaskQCExtractor(session_path, lazy=True, one=one)
+            # Extract extra datasets required for QC
+            self.extractor.data = dsets
+            self.extractor.extract_data()
```
Comment (k1o0) [inline task_qc_viewer/task_qc.py:55]: Add `partial=True`, otherwise the same data (`dsets` will be re-extracted.

**Pull request merged**